ONSdigital/design-system
# Align checkbox indent line with the checkboxes

### What is the context of this PR?

Fixes #3384 

This PR adjusts the alignment of the checkbox indent line, ensuring it is centered with the checkboxes. The left margin on the line has been increased for better alignment

### How to review this PR

- Check that all visual changes show the intended alignment.
- Run the checkbox component locally and confirm that the indent line is now properly centered with the checkboxes

### Checklist

This needs to be completed by the person raising the PR.
